roles/gateway_processing_node/tasks/main.yml

```diff
@@ -37,13 +33,24 @@
         - { from: '^cluster.host', to: 'cluster.host={{ cluster_host }}' }
         - { from: '^#cluster.pass', to: 'cluster.pass={{ cluster_pass }}' }
         - { from: '^configure.db', to: 'configure.db={{ configure_db|lower }}' }
+        - { from: '^database.type', to: 'database.type={{ database_type }}' }
```

Comment:
All these values could have been customized and here it ensures we capture all of them.